doc/getting-started/connections.rst

```diff
@@ -0,0 +1,75 @@
+Connections
+===========
+
+PyVista is a powerful tool that researchers can harness to create compelling,
+integrated visualizations of large datasets in an intuitive, Pythonic manner.
+
+PyVista is used across science and engineering disciplines by a diverse
+community to visualize data/models in 3D, generate publication-quality
+illustrations, automate analysis workflows, and build custom applications
+leveraging PyVista's 3D capabilities.
+
+Here are a few open-source projects that leverage PyVista:
+
+
+Engineering
+-----------
+
+* pyleecan_: Electrical engineering open-source software providing a user-friendly, unified, flexible simulation framework for the multiphysic design and optimization of electrical machines and drives.
+* pymapdl_: Pythonic interface to Ansys MAPDL.
+* pyprocar_: A Python library for electronic structure pre/post-processing.
+* AeroSandbox_: Plug-and-play analysis tools for aerodynamics, propulsion, structures, trajectory design, and much more.
+* Ptera_: Software for analyzing flapping-wing flight.
+
```

Comment:
Feel free to add PyDPF-Core and PyAEDT if you think that's not too many PyAnsys projects. You could also just reference the PyAnsys org.